A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -853,11 +853,11 @@ class CDUPerformancePage {
 
         mcdu.setTemplate([
             ["APPR[color]" + titleColor],
-            ["QNH", "FINAL", "FLP RETR"],
+            ["QNH", "FINAL", "FLP RETR{sp}"],
             [qnhCell + "[color]cyan", finalCell + "[color]green", "F=" + flpRetrCell + "[color]green"],
-            ["TEMP", "MDA", "SLT RETR"],
+            ["TEMP", "MDA", "SLT RETR{sp}"],
             [tempCell + "°[color]cyan", mdaCell + "[color]cyan", "S=" + sltRetrCell + "[color]green"],
-            ["MAG WIND", "DH", "CLEAN"],
+            ["MAG WIND", "DH", "{sp}{sp}CLEAN"],
             [magWindHeadingCell + "°/" + magWindSpeedCell + "[color]cyan", dhCell + "[color]cyan", "0=" + cleanCell + "[color]green"],
```

Comment:
```suggestion
            [magWindHeadingCell + "°/" + magWindSpeedCell + "[color]cyan", dhCell + "[color]cyan", "O=" + cleanCell + "[color]green"],
```

Let's fix this here, because it's really obvious now 😅 Would fix #1823